pkg/config/config_template.yaml

```
@@ -4149,10 +4149,12 @@ api_key:
   ##
   ## Example:
   ## - conditions:
-  ##     service_name: exampleService
+  ##     command_line_regex: command_line_regex: "exampleService"
```

I think this might be a copy/paste error - `command_line_regex: ` appears twice.

```suggestion
  ##     command_line_regex: executable_name.exe
```